semantic.json

```diff
@@ -1,5 +1,5 @@
 {
-  "base": "public/semantic/",
+  "base": "public/semantic",
```

Comment:
Let's start removing stuff you're not using, including old Blade templates. 😄   
It's confusing to me to hunt stuff down when old stuff lies around, recovering old stuff is what version control is for, don't be afraid to delete things!